app/models/pglogical_subscription.rb

```diff
@@ -263,4 +261,8 @@ def with_remote_connection
       yield conn
     end
   end
+
+  def subscription_status
```

Comment:
Oh this is confusing. There is a class method with the same name that isn't related to this instance method ... Naming is hard.